src/sentinel.c

```diff
@@ -457,6 +461,18 @@ dictType renamedCommandsDictType = {
     NULL                       /* allow to expand */
 };
 
+/* Dict for for case-insensitive search using null terminated C strings.
+ * The keys stored in dict are sds though. */
+dictType stateSetDictType = {
+    distCStrCaseHash,           /* hash function */
```

Comment:
@hwware Why do you use null-terminated strings here where the value is really an sds?

Reply:
@yossigo Yes, you are right. So in the dict, I choose not changing the char* to sds, directly store as char*.  And I realize that the PR https://github.com/redis/redis/pull/6891 should update codes like this as well.  Thanks for mentioning this.